lib/bignum.toit

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import encoding.hex
+
+BIGNUM_ADD_ ::= 0
+BIGNUM_SUB_ ::= 1
+BIGNUM_MUL_ ::= 2
+BIGNUM_DIV_ ::= 3
+BIGNUM_MOD_ ::= 4
+
+class Bignum:
+  sign_/bool := ?
+  limbs_/ByteArray := ?
+
+  constructor .sign_/bool .limbs_/ByteArray:
```

Comment:
Perhaps this constructor should be named and private, and `trans_other_` should be the unnamed constructor.

Reply:
Perhaps the `string` one should be unnamed constructor, because it use by user and SSL certification directly.